airbyte-commons-worker/src/main/java/io/airbyte/workers/process/DockerProcessFactory.java

```diff
@@ -169,6 +173,31 @@ public Process create(final String jobType,
     }
   }
 
+  /**
+   * !! ONLY FOR DEBUGGING, SHOULD NOT BE USED IN PRODUCTION !! If you set the DEBUG_CONTAINER_IMAGE
+   * environment variable, and it matches the image name of a spawned container, this method will add
+   * the necessary params to connect a debugger. For example, to enable this for
```

Comment:
It could be worth adding this information to some dev docs if you know where they are, quick search sent me through all the specific connector's readme.

Reply:
I'll poke around and see if I can find a good docs home